openlibrary/macros/ReadButton.html

```diff
@@ -12,14 +12,14 @@
 $ stream_url = "/borrow/ia/%s?ref=ol" % ocaid
 
 <div class="cta-button-group">
-  <a href="$(stream_url)" title="$title" id="$(action)_ebook"
+  <a href="$(stream_url)" title="$title" 
      $if loan:
        data-userid="$(loan['userid'])"
      $elif borrow:
        data-ol-link-track="CTAClick|Borrow"
      $else:
        data-ol-link-track="CTAClick|Read"
-     class="cta-btn cta-btn--available">$label</a>
+     class="cta-btn cta-btn--available $(action)_ebook">$label</a>
```

Comment:
cta-btn--$(action) would be better here as it keeps with the BEM syntax we are using for the other classes.